README.md

```diff
@@ -167,6 +167,12 @@ A responsive [THREE.PerspectiveCamera](https://threejs.org/docs/index.html#api/e
 
 A responsive [THREE.OrthographicCamera](https://threejs.org/docs/index.html#api/en/cameras/OrthographicCamera) that can set itself as the default.
 
+#### useCameraShake
+
+[![](https://img.shields.io/badge/-codesandbox-blue)](https://codesandbox.io/s/simplex-camera-shake-j6oob)
+
+A hook for applying a configurable camera shake effect.
```

Comment:
this should have an example with the config roughly explaing config opts, a bit like `useAspect`

Reply:
Added a simple example, just explaining the main config options. I tried to keep it pretty breif, wasn't sure how in depth I should go in the readme. I didn't go into some of the more advanced examples, but there is more info explaining those in the sandbox. More/Less in the readme, just lmk.